settings.gradle.kts

```diff
@@ -19,10 +19,10 @@ pluginManagement {
     }
 
     plugins {
-        id("com.android.application") version ("7.1.0")
-        id("com.android.library") version ("7.1.0")
-        id("org.jetbrains.kotlin.android") version ("1.6.10")
-        id("org.jetbrains.dokka") version ("1.6.10")
+        id("com.android.application") version ("7.2.1")
+        id("com.android.library") version ("7.2.1")
+        id("org.jetbrains.kotlin.android") version ("1.7.10")
+        id("org.jetbrains.dokka") version "1.7.0"
```

Comment:
Can you add the parenthesis back around the version number here (for consistency)?